README.md

```diff
@@ -1,285 +1,78 @@
-<a name="readme-top"></a>
+# BLOG APP
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
 
-Modify this file to match your project and remove sections that don't apply.
+> This is a budget app that helps yoou keep track of your expenses
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
 
-After you're finished please remove all the comments and instructions!
--->
+## demo video
+https://www.loom.com/share/3802e678ac0a4118965be1f5736fa116
```

Comment:
### The video you sent is missing two crucial requirements:
- [ ] The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.